srcjs/init_shiny.js

```diff
@@ -207,6 +207,15 @@ function initShiny() {
   }
   exports.initializeInputs = initializeInputs;
 
+  function getIdFromEl(el) {
+    var $el = $(el);
+    var bindingAdapter = $el.data("shiny-output-binding");
+    if (!bindingAdapter)
+      return null;
+    else
+      return bindingAdapter.getId();
```

Comment:
No, it's intentional: https://github.com/rstudio/shiny/pull/918/files#diff-1f04015072adf4e6718b3ea09773dd7dR2945